tests/scaffoldings/dotd/custom/src/addons.yaml

```diff
@@ -3,6 +3,10 @@ other-doodba/odoo/src/private:
   - dummy_addon
   - product
 
+odoo/addons:
+  - "*"
+  - "!hr_payroll"
```

Comment:
Answering here to https://github.com/Tecnativa/doodba/pull/210/files#r269939304, I mean that you please add a negative glob here.

If users can use globs to include addons, it is expected that they can use it also to exclude. The code you pushed doesn't support exclusion globs, nor does it know what to do if there are only negative globs, so that's why I ask you to add a test with globs and make it pass (to have [TDD](https://en.wikipedia.org/wiki/Test-driven_development)).

A good example:

```yaml
odoo/addons:
  - "!hr_payroll*"
```

Then, test that:

- i.e. `sale` addon is present (because of the implicit `"*"` (only implicit in the special `odoo/addons` and `private` sections)).
- `hr_payroll` and `hr_payroll_account` are absent (because it matches the negative glob).

Reply:
OK, I understand you now, you was thinking on use negative globs as exclusion mechanism. Will try!